app/models/company.rb

```diff
@@ -58,6 +58,8 @@ def last_char_star?
   end
 
   def refresh_search_suggestions
+    # It feels weird that you're calling a class method directly on a concern.
+    # Could this be a service object?
```

Comment:
@tylerwillingham, Thanks for the articles and explanations. Definitely gives me some ideas as to placing some of these concerns.